.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Git hub continuos integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v2
```

Comment:
To fix the failing job you can update the version, to use v4, or add a parameter in the with section, to specify the config file:
`config-file: cypress.config.js`

Reply:
I've update the version to v4 and solve the issue